### PR TITLE
Issue 312: Corrects the capitalisation of 'playbook'

### PR DESCRIPTION
**Pull request by bredamc**
Proposed fix for Issue 312.

### CODE REVIEW

**Review by bburt-rh — DISMISSED** [review]

**Comment by bburt-rh** [comment]
lgtm

**Review by mportman12 — DISMISSED** [review]
LGTM

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc [hunk 1]
```diff
@@ -179,7 +179,7 @@
 [discrete]
 [[ceph-ansible]]
 ==== image:images/yes.png[yes] ceph-ansible (noun)
-*Description*: In Red Hat Ceph Storage, `ceph-ansible` is a utility that provides Ansible playbooks for installing, managing, and upgrading the Ceph Storage Cluster. Always mark it correctly (`ceph-ansible`).
+*Description*: In Red Hat Ceph Storage, `ceph-ansible` is a utility that provides Ansible Playbooks for installing, managing, and upgrading the Ceph Storage Cluster. Always mark it correctly (`ceph-ansible`).
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:182]: Always mark it correctly (`ceph-ansible`).

> Always mark it correctly: `ceph-ansible`.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc [hunk 2]
```diff
@@ -788,13 +788,13 @@ image repository contains one or more tagged images.
 [discrete]
 [[control-node]]
 ==== image:images/yes.png[yes] control node (noun)
-*Description*: Any machine with Ansible installed. You can run commands and playbooks, invoking /usr/bin/ansible or /usr/bin/ansible-playbook, from any _control node_. You can use any computer that has Python installed on it as a control node - laptops, shared desktops, and servers can all run Ansible. However, you cannot use a Windows machine as a control node. You can have multiple control nodes.
+*Description*: Any machine with Ansible installed. You can run commands and playbooks, invoking `/usr/bin/ansible` or `/usr/bin/ansible-playbook`, from any _control node_. You can use any computer that has Python installed on it as a control node: laptops, shared desktops, and servers can all run Ansible. However, you cannot use a Windows machine as a control node. You can have multiple control nodes.
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:791]: I think the last sentence, _You can have multiple control nodes_, should be moved elsewhere as it feels out of place with the Windows sentence that comes before it. The can > cannot > can paragraph construction is a bit off. Perhaps finish with the Windows sentence?

**Review by dfitzmau — APPROVED** [review]

**Review by mportman12 — APPROVED** [review]
LGTM

**Review by sbmetz — APPROVED** [review]
LGTM

**Pull request merged**